.gitignore

```diff
@@ -128,3 +131,18 @@ dist
 .yarn/build-state.yml
 .yarn/install-state.gz
 .pnp.*
+
+# package-lock.json
+package-lock.json
```

Comment:
It's not necessary, we need this file. Review: https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5

Reply:
Oh, is ok